01-gabbar/02-stakater-nordmart-review-ui/02-stage/values.yaml

```diff
@@ -8,3 +8,7 @@ stakater-nordmart-review-ui:
         repository: nexus-docker-stakater-nexus.apps.devtest.vxdqgl7u.kubeapp.cloud/stakater-nordmart-review-ui
         tag: 1.0.2
       imagePullSecrets: nexus-docker-config-forked-devtest
+      env:
+        REVIEW_API:
+          # This is hardcoded!
+          value: "https://review-gabbar-prod.apps.stage.2cc6dtsv.kubeapp.cloud/"
```

Comment:
Why it has name -prod?

Reply:
because there are 2 namespaces stage, and prod

https://github.com/stakater/gitops-config-template/blob/main/03-sre/02-stage-prod/tenant-operator-config/tenants/gabbar.yaml

Reply:
but why same url on both environments?